var/spack/repos/builtin/packages/py-exarl/package.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
```

Comment:
@eugeneswalker I believe the issue with the style is that the last header date should read `2023` not 2021.

Reply:
Just pushed cd4a4c28ecdebfaaa26eb1e0f9d88f03daad4551

Reply:
Sweet! Looks good to me!